packages/storybook/package.json

```diff
@@ -41,9 +55,9 @@
     "postcss-rem-to-responsive-pixel": "^5.1.1",
     "postcss-simple-vars": "^6.0.3",
     "serve": "^13.0.4",
-    "storybook": "^7.6.1",
-    "typescript": "4.9.5",
-    "vite": "^4.4.9",
+    "storybook": "^8.1.11",
+    "typescript": "5.3.3",
```

Comment:
Not sure if this will give issues down the line as it doesn't match the rest of monorepo? (I prefer all packages to have the same version)

Reply:
Okay, reverting ts and svgr!